src/dlgprefshoutcast.cpp

```diff
@@ -212,31 +214,65 @@ void DlgPrefShoutcast::slotResetToDefaults() {
 }
 
 void DlgPrefShoutcast::slotUpdate() {
-    enableLiveBroadcasting->setChecked((bool)m_pConfig->getValueString(
-        ConfigKey(SHOUTCAST_PREF_KEY,"enabled")).toInt());
+    enableLiveBroadcasting->setChecked(m_pShoutcastEnabled->toBool());
+
+    // Don't let user modify information if
+    // sending is enabled.
+    if(m_pShoutcastEnabled->toBool() == true) {
```

Comment:
Just habit of my to write this open because I'm lazy to think is it true or false but you are correct..